src/actions/Material.js

```diff
@@ -61,3 +61,12 @@ export function setBoundaryConditionsForOneMaterial(config) {
         ...config,
     }
 }
+
+export const MATERIALS_SET_IS_NON_PERIODIC_FOR_ONE = 'MATERIALS_SET_IS_NON_PERIODIC_FOR_ONE';
+
+export function setIsNonPeriodicConditionsForOneMaterial(config) {
+    return {
+        type: MATERIALS_SET_IS_NON_PERIODIC_FOR_ONE,
+        ...config,
+    }
+}
```

Comment:
We need the corresponding reducer now to set the flag for the material in question

Reply:
Please see [src/reducers/Material.js ](https://github.com/Exabyte-io/materials-designer/pull/29/files#diff-07117e6feb13357543259d645dcae43645f34789d559cfebdcd62a99e17c3604)